cinema/models.py

```diff
@@ -27,6 +27,10 @@ class Actor(models.Model):
     first_name = models.CharField(max_length=255)
     last_name = models.CharField(max_length=255)
 
+    @property
+    def full_name(self):
+        return f"{self.first_name} {self.last_name}"
+
     def __str__(self):
         return self.first_name + " " + self.last_name
```

Comment:
Seems like str method here is doing exact same thing as the full_name method. Dunno if it is a mistake, but I'd have changed the logic a bit in my solution.